app/src/main/java/com/example/oddhours/database/TableJobs.kt

```diff
@@ -70,42 +66,23 @@ class TableJobs {
     /**
      * getAllJobID - returns a list of all the Job IDs in the database
      */
-    fun getAllJobID(): List<Int>{
-        val db = DatabaseHelper.database
-        var listOfJobIds = mutableListOf<Int>()
+    fun getAllJobs(): List<JobModel>{
+        var listOfJobIds = mutableListOf<JobModel>()
         val res = db!!.rawQuery(
-            "SELECT ${DatabaseHelper.job_ID_COL_1} FROM ${DatabaseHelper.jobTable}",null
+            "SELECT * FROM ${DatabaseHelper.jobTable}",null
         )
+        var jobModel: JobModel
         if(res.count != 0){
             while(res.moveToNext()){
-                listOfJobIds.add(res.getInt(0))
+                jobModel = JobModel(res.getInt(0), res.getString(1), res.getString(2))
+                listOfJobIds.add(jobModel)
             }
             return listOfJobIds
         }
         return listOfJobIds
```

Comment:
i'm just wondering here when `res.count = 0`, we return a `listOfJobIds` that is only initialized. is it possible that accessing this empty list can cause an error if we don't properly check for that scenario first? i'm not sure so just checking

Reply:
So, you know how we have that check at the beginning, if no jobs are added we show the text No Job, instead of the recyclerView, we can apply the same logic to the ShiftsList... where we check if any jobs exist first if not, just show a text "No Jobs have been added yet"

Reply:
right, i see i see. makes sense then